src/style-library/components/atoms/InheritancePill.scss

```
@@ -0,0 +1,41 @@
+.kadence-blocks-style-library__inheritance-pill {
+	display: inline-flex;
+	align-items: center;
+	box-sizing: border-box;
+	// The two variants must measure the same height even though only one of them draws a border:
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)